src/createServer.js

```diff
@@ -1,3 +1,37 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const convertToCase = require('./convertToCase/convertToCase').convertToCase;
+const getErrors = require('./getErrors').getErrors;
+
+function createServer() {
+  const server = http.createServer((request, response) => {
+    const currentURL = new URL('htto://' + request.url);
+
+    response.setHeader('Content-Type', 'application/json');
+
+    const requestText = currentURL.pathname.slice(1);
+    const requestCase = currentURL.searchParams.get('toCase');
+    const errors = getErrors(requestText, requestCase);
+
+    if (!errors.errors.length) {
```

Comment:
You can return just an array of errors from `getErrors` because it looks a bit strange